.coderabbit.yaml

```
@@ -0,0 +1,29 @@
+  eslint:
+    enabled: true   # JS/TS 코드 분석
+  gitleaks:
+    enabled: true   # 비밀키/토큰 유출 검사
+  markdownlint:
+    enabled: true   # Markdown 문서 스타일 검사
+
+knowledge_base:
+  code_guidelines:
+    enabled: true   # 팀 코드 규칙 문서 반영
+    filePatterns:
+      - ".github/instructions.md"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)